experimental/origin-admin/src/queries/Fragments.js

```diff
@@ -20,7 +20,7 @@ export default {
   },
   Listing: {
     basic: gql`
-      fragment basicListingFields on ListingResult {
+      fragment basicListingFields on Listing {
```

Comment:
Why are those changes part of this PR ? they seem unrelated...

Reply:
tnx for catching this, yes not part of this PR